.github/workflows/deploy.yml

```diff
@@ -70,5 +70,5 @@ jobs:
 
       - name: Delete previous Docker image(s)
         run: |
-          gcloud container images list-tags $IMAGE_NAME:$REF_NAME --filter='-tags:*' --format='get(digest)' --limit=unlimited | \
```

Comment:
Here you removed `ref_name` but at line 56 (and few before) you changed it to `tag_name`- is it a typo or it should be like that?